src/schema/objects/metadata.yaml

```diff
@@ -2825,6 +2825,57 @@ ScatterFraction:
     type: number
     minimum: 0
     maximum: 100
+ScreenDistance:
+  name: ScreenDistance
+  display_name: Screen Distance
+  description: |
+    Distance between the participant's eye and the screen. If no screen was used, use `n/a`.
+  anyOf:
+    - type: number
+      unit: cm
+    - type: string
+      enum:
+        - n/a
+ScreenRefreshRate:
+  name: ScreenRefreshRate
+  display_name: Screen Refresh Rate
+  description: |
+    Refresh rate of the screen (when used), in Hertz (equivalent to frames per second, "FPS").
+  type: number
+  unit: Hz
+ScreenResolution:
+  name: ScreenResolution
+  display_name: Screen Resolution
+  description: |
+    Screen resolution in pixel
+    (for example `[1920, 1200]` for a screen of 1920-width by 1080-height pixels),
+    if no screen use `n/a`.
+  anyOf:
+    - type: array
+      items:
+        type: integer
+        minItems: 2
+        maxItems: 2
+    - type: string
+      enum:
+        - n/a
+ScreenSize:
+  name: ScreenSize
+  display_name: Screen Size
+  description: |
+    Screen size in cm, excluding potential screen borders
+    (for example `[47.2, 29.5]` for a screen of 47.2-width by 29.5-height cm),
+    if no screen use `n/a`.
+  anyOf:
+    - type: array
+      items:
+        type: number
+        unit: cm
```

Comment:
Especially here `m` would much better unit IMHO... and as such I would prefer all of them to be just meters

Reply:
why would meter be much better for screen size? screens are usually smaller than one meter, aren't they?

Reply:
I think meter would be better because it is the SI base unit, i.e., the unit without prefix, as recommended [here](https://bids-specification.readthedocs.io/en/stable/appendices/units.html). That also aligns with the related derived units (and measurements), such as Luminance (cd/m^2).

Reply:
either way I don't feel too strongly about this, though relying on pure unprefixed SI as much as possible has its merits.

@mszinte any opinion?